android/build.gradle

```diff
@@ -143,8 +143,8 @@ dependencies {
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
   implementation 'com.android.support:appcompat-v7:28.0.0'
 
-  debugImplementation files('libs/community-pass-library-v2.4.0.aar')
-  releaseCompileOnly files('libs/community-pass-library-v2.4.0.aar')
+  debugImplementation files('libs/community-pass-library-v2.5.1.aar')
```

Comment:
Are these changes made on the document as well? Getting Started and Contributin.md?
\

Reply:
Resolved [here](https://github.com/Mastercard/community-pass-react-native-wrapper/pull/17/commits/35b625b1fc64c7eaed4282b21819895eedb5bb21)